README.md

```diff
@@ -7,17 +7,17 @@ particularly in the fields of oceanography and meteorology.
 There are currently four courses:
 
 ### An introduction to numpy
-3.5 hours &mdash; depends on a basic Python background
-http://nbviewer.ipython.org/github/SciTools/courses/blob/master/course_content/notebooks/numpy_intro.ipynb?create=1
+3.5 hours &mdash; depends on a basic Python background  
```

Comment:
Given that I've made another comment I'll just highlight this too - you've trailing whitespace at the end of each of these lines. If you need to make another code change then perhaps tidy these up too, but if not then let's get it done in a follow-up PR.

Reply:
The trailing double whitespace starts a [linebreak](https://gist.github.com/shaunlebron/746476e6e7a4d698b373) for the URL.\
I could use a more explicit version (`/` or `</br>`) if you would prefer.